frontend/views/containers/chatroom/SendArea.vue

```diff
@@ -600,9 +597,9 @@ export default ({
         const lastDotIndex = name.lastIndexOf('.')
         return lastDotIndex === -1 ? '' : name.substring(lastDotIndex).toLowerCase()
       }
-      const attachmentsExist = Boolean(this.ephemeral.attachment.length)
+      const attachmentsExist = Boolean(this.ephemeral.attachments.length)
```

Comment:
(This is something I missed in my own work but), `attachmentsExist` here can be replaced with the computed prop `this.hasAttachments` defined above.